sos/report/plugins/microshift_ovn.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2023 Pablo Acevedo <pacevedo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class MicroShiftOVN(Plugin, RedHatPlugin):
+    """This plugin is used to collect MicroShift 4.x OVN logs.
+    """
+    short_desc = 'MicroShift OVN'
+    plugin_name = "microshift_ovn"
+    profiles = ('microshift',)
+
+    def setup(self):
+        self.add_copy_spec([
+            '/etc/openvswitch/conf.db',
+            '/etc/openvswitch/default.conf',
+            '/etc/openvswitch/system-id.conf'])
+        
+        self.add_cmd_output(['ovs-appctl -t /var/run/ovn/ovnnb_db.ctl ' +
+            'coverage/show',
+            'ovs-appctl -t /var/run/ovn/ovnsb_db.ctl coverage/show',
+            'ovs-appctl -t /var/run/ovn/ovnnb_db.ctl memory/show',
+            'ovs-appctl -t /var/run/ovn/ovnsb_db.ctl memory/show',
+            'ovs-appctl -t /var/run/ovn/ovnnb_db.ctl ' +
+            'ovsdb-server/get-db-storage-status OVN_Northbound',
+            'ovs-appctl -t /var/run/ovn/ovnsb_db.ctl ' +
+            'ovsdb-server/get-db-storage-status OVN_Southbound',
+            'ovs-appctl -t /var/run/ovn/ovnnb_db.ctl ' +
+            'ovsdb-server/sync-status',
+            'ovs-appctl -t /var/run/ovn/ovnsb_db.ctl ' +
+            'ovsdb-server/sync-status'])
```

Comment:
The defacto sos-style for this would be to define a list of subcmds and then use list comprehension like below so that it's easier to read and extend.

```python
ovs_cmd = 'ovs-appctl -t /var/run/ovn/'
subcmds = [
    'coverage/show',
    'memory/show',
    [...]
]

for ovn_db in ['ovnnb_db.ctl', 'ovnsb_db.ctl']:
    self.add_cmd_output([
        f"{ovs_cmd}{ovn_db} {cmd}" for cmd in subcmds
    ])
```

As a bonus, it helps with the PEP8 requirements.

Reply:
Done.